### PR TITLE
Fix mgpu executor

### PR DESCRIPTION
**Pull request by yhwen**
Fixed the PTMultiProcessExecutor error. Create the fobs_initialize() for all the processes.

### CODE REVIEW

**Review by yanchengnv — APPROVED** [review]
LGTM

File: nvflare/private/fed/app/client/worker_process.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def main():
     if os.path.exists(restart_file):
         os.remove(restart_file)
 
+    fobs_initialize()
```
[the remark this answers is not in the document]
Comment (yhwen) [reply nvflare/private/fed/app/client/worker_process.py:96]: No, this is not only for the check_parent_alive(). This is the client app running process. It needs to initialize the fobs.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks LGTM

File: nvflare/apis/utils/decomposers/flare_decomposers.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def supported_type():
         return Workspace
 
     def decompose(self, target: Workspace) -> Any:
-        return [target.root_dir, target.name, target.config_folder]
+        return [target.root_dir, target.site_name, target.config_folder]
```
Comment (YuanTingHsieh) [inline nvflare/apis/utils/decomposers/flare_decomposers.py:114]: Note this change is the same as in #914

Comment (yhwen) [reply nvflare/apis/utils/decomposers/flare_decomposers.py:114]: that's fine. PR merge will take care of this.

**Pull request merged**